src/validations/noFakeLists.js

```diff
@@ -0,0 +1,29 @@
+
+/**
+ * @param {Object} [options = {}] - Options for the validation
+ * @param {RegExp} [options.regex =  /\n\s*[-*+–—]/] - Validation regex
+ * @param {String} [options.message = "This looks like a list, but it is plain text. Use the bulleted list option."] - Message to display when the validation triggers
+ */
+
+function noFakeLists(blocks, options = {}) {
+  const fakeListBlocks = (blocks || []).filter(
+    block =>
+      block._type === 'block' &&
+      block.children.some(blockPart =>
+        blockPart._type === 'span' &&
+        blockPart.text.match(options.regex || /\n\s*[-*+–—]/)
```

Comment:
In theory, this could be a little more costly than declaring the regular expression outside of the loop because a new regular expression instance gets created for every single node. V8 and other JavaScript engines are particularly good at optimizing for cases like this so it shouldn’t really matter in practice though.